### ISSUE TITLE
bring back Muller2008 and add Zahirovic2022 model

### ISSUE THREAD

**Issue by michaelchin** [GPlates/gplately#284]
waiting for DM's confirmation...

**Comment by brmather** [comment]
Might as well bring them back if people want them. I think it's better to have an exhaustive list of plate models for people to choose from. Better to have too many plate models than too few...

**Comment by michaelchin** [comment]
> Might as well bring them back if people want them. I think it's better to have an exhaustive list of plate models for people to choose from. Better to have too many plate models than too few...

Yes, this is a good idea. I may need a RA to help me with this. To request adding models, go to https://github.com/GPlates/gplately/discussions/217

**Comment by michaelchin** [comment]
I believe the consensus is bring back Muller2008 and add Zahirovic2022. 

Not sure if Zahirovic2022 is on Zenodo. 

Not sure if we should put Muller2008 onto Zenodo.

Ask these questions later when the wind's not blowing in the East(this is a bad attempt at a joke out of Bleak House which is torturing me now).

**Comment by michaelchin** [comment]
find Muller2008 at https://earthbyte.org/webdav/ftp/Data_Collections/Muller2008/

get Zahirovic2022 from GeoData bundle

**Comment by michaelchin** [comment]
Is Zahirovic2022 on Zenodo?

**Comment by michaelchin** [comment]
> Is Zahirovic2022 on Zenodo?

The Zahirovic_etal_2022 model  files are part of this zenodo archive


https://zenodo.org/records/4729046

**Comment by michaelchin** [comment]
This has to wait. I am on pygmt and web service right now.

**Comment by brmather** [comment]
> This has to wait. I am on pygmt and web service right now.

A note on pygmt — it supports geopandas data frames, which is basically what we obtain using the get_ methods in PlotTopologies. So hopefully it is not too much work for you 🤞

**Comment by michaelchin** [comment]
> > This has to wait. I am on pygmt and web service right now.
> 
> A note on pygmt — it supports geopandas data frames, which is basically what we obtain using the get_ methods in PlotTopologies. So hopefully it is not too much work for you 🤞

Thanks Ben. I will seek help when the time comes.

**Comment by michaelchin** [comment]
I believe that I have brought back Muller2008 and  added Zahirovic2022. Just need to update the documentation so that people are not confused.
